src/components/Header.tsx

```diff
@@ -106,7 +109,14 @@ const HeaderLinks = () => {
 const Header = (props: BoxProps) => {
   const isClientSide = useIsClientSide()
   return (
-    <Box as="header" bg="gray.700" color="white" zIndex="banner" {...props}>
+    <Box
+      as="header"
+      zIndex="banner"
+      {...props}
+      // TODO: borderTop should be 4px when on iPhones
```

Comment:
You can achieve that with [Responsive Styles using object syntax](https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax):

`borderTop={{ base: '4px', md: '8px' }}`

Reply:
Is that `md` breakpoint the same as the iPhone 11 size in our design?

> We've chosen to start with a design for **iPhone 11 Pro / X** which is:
> 
>     * `375px` wide
> 
>     * `812px` high

I'm bad at converting Chakra UI em units to px...

https://chakra-ui.com/docs/theming/theme#breakpoints

Reply:
Also, while we're discussing the `borderTop` property: do we want the `borderTop` to be on the `<header>` tag or on the `<body>` tag? I'm not sure which option is best.

I couldn't find the correct point to adjust the `<body>` tag in the code, so that's why I put the colored bar on the `<header>`.... 😄

Reply:
> Is that `md` breakpoint the same as the iPhone 11 size in our design?

I don't think it's _exactly_ the iPhone 11 size, but we need to define this by mobile, tablet, regular desktop, and large desktop rather than by the exact size of one device.



> Also, while we're discussing the `borderTop` property: do we want the `borderTop` to be on the `<header>` tag or on the `<body>` tag? I'm not sure which option is best.
> 
> I couldn't find the correct point to adjust the `<body>` tag in the code, so that's why I put the colored bar on the `<header>`.... 😄

Doing this in our Header is the perfect place to do so. The `body` is outside Next.js and Chakra scope, so you need to style it through [Chakra global's style](https://github.com/Belco90/octoclairvoyant/blob/main/src/customTheme.ts#L124)

Reply:
> I don't think it's _exactly_ the iPhone 11 size, but we need to define this by mobile, tablet, regular desktop, and large desktop rather than by the exact size of one device.

I think the `sm` breakpoint matches closest, so that's what I've used in my new commits.

> Doing this in our Header is the perfect place to do so. The `body` is outside Next.js and Chakra scope, so you need to style it through [Chakra global's style](https://github.com/Belco90/octoclairvoyant/blob/main/src/customTheme.ts#L124)

Whew, so it's in the correct place right now! 😄